### DISCUSSION TITLE
Milvus 2.2集群版部署问题咨询

### DISCUSSION THREAD

**Discussion by RongHouchao** [milvus-io/milvus#22022] [Q&A and General discussion]
**1）项目部署时已经有了etcd和minio，部署milvus时需要复用之前的集群里的etcd和minio，这个怎么操作？我看values.yaml里没有配置etcd和minio服务地址的地方。**
![image](https://user-images.githubusercontent.com/117245915/217142137-587a50a6-9c2c-493f-bdf1-b16c76ba8ea5.png)
![image](https://user-images.githubusercontent.com/117245915/217142225-a7cd64a6-9178-46e5-9607-3ad3442e3e89.png)

**2）pulsar-bookie和pulsar-zookeeper的pod实例启动个数默认是3，我想都修改成1个实例，values.yaml没找到这个配置。**
![image](https://user-images.githubusercontent.com/117245915/217142439-71e0e955-e27a-4d53-b032-95c853267139.png)
![image](https://user-images.githubusercontent.com/117245915/217142671-7470aca6-9aa5-4ba0-891b-208d08daf26d.png)

**3）目前Milvus默认使用的是NFS作为持久化的方式，如果使用local pv的方式部署，没看到这方面的文档。**

**Comment by LoveEachDay** [comment]
1: For external minio and etcd, you can enable externalEtcd and externalS3. Follow the instructions here: https://milvus.io/docs/deploy_etcd.md and https://milvus.io/docs/deploy_etcd.md

2: pulsar bookie require 2 pods at least. For minimum installation, you can set like this: `--set pulsar.bookkeeper.replicaCount=2 --set pulsar.zookeeper.replicaCount=1`

3. By default, we use `storageclass` to automatically provision pv and pvc. You can manually create the pv and pvc before you install milvus.
